collector-bitmovin-player/src/main/java/com/bitmovin/analytics/bitmovin/player/BitmovinSdkAdapter.java

```diff
@@ -247,10 +270,17 @@ public EventData createEventData() {
     @Override
     public void release() {
         playerIsReady = false;
+        if(this.segmentTrackingAdapter != null) {
```

Comment:
Long term we should build a analytics event-bus so the feature can hook up to a `release` event and we don't have to manually release each feature in the main adapter

Reply:
Could you check the last two commits for that? It's a bit ugly right now as the eventbus is really simple - but it should do what we want and for this beta release should be fine I guess?
I'd create a separate issue to make a streamlined event bus here.